source/renderer/app/components/sidebar/Sidebar.js

```diff
@@ -33,16 +33,16 @@ type Props = {
 
 export type SidebarMenus = {
   wallets: ?{
-    items: Array<SidebarWalletType>,
-    activeWalletId: ?string,
-    actions: {
+    items?: Array<SidebarWalletType>,
```

Comment:
if `wallets` inside the sidebar menu is not empty then items and actions and activeWalletId parameters are required. Do not change types just to make flow happy. It is safer and more convenient to check the full flow path and fix params you are passing.

Reply:
Fixed